yellowbrick/features/decomposition.py

```diff
@@ -117,8 +131,14 @@ def transform(self, X):
         return self.pca_features
 
     def draw(self):
+        if self.ax is None:
+            self.ax = self.gca()
```

Comment:
Lines 134 and 135 are no longer necessary, the `Visualizer.ax` property ensures that if `self.ax` is None, then either the current axes object is fetched or a new axes is created.